cmd/minikube/cmd/start.go

```diff
@@ -908,11 +934,6 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 	validateCPUCount(drvName)
 
 	if cmd.Flags().Changed(memory) {
-		req, err := util.CalculateSizeInMB(viper.GetString(memory))
```

Comment:
Why get rid of this check?

Reply:
because it was doing it twice, and was priniting the advice two times.

Reply:
it was move to line 397 to be done all the time, instead of when check change memory

Reply:
also was moved to 397 because at that time we have the suggested auto calucated memory. we will need to verify that one is allowed too